pages/[id]/index.js

```diff
@@ -13,6 +15,7 @@ function SnippetDetailsPage({ onToggleFavorite, favorites }) {
     try {
       await fetch(`/api/snippets/${id}`, { method: "DELETE" });
       mutate("/api/snippets");
+      toast.success("Deleted successfully!");
```

Comment:
In your edit component you created a function called notify, and then call notify when it is time to show the popup. 

i.e. 

``` js
const notify = () => toast.success("Edited snippet successfully.");
```

Here you directly call the `success` method on the `toast` object. Both work, but it might make sense to go with a consistent approach.